packages/desktop-client/src/components/mobile/accounts/Account.jsx

```diff
@@ -138,43 +133,50 @@ export function Account(props) {
             tables.includes('category_mapping') ||
             tables.includes('payee_mapping')
           ) {
-            paged?.run();
+            paged.current?.run();
           }
 
           if (tables.includes('payees') || tables.includes('payee_mapping')) {
-            actionCreators.getPayees();
+            dispatch(actions.getPayees());
           }
         }
       });
 
       await fetchTransactions();
 
-      actionCreators.markAccountRead(accountId);
+      dispatch(actions.markAccountRead(accountId));
     }
 
     setUpAccount();
 
     return () => unlisten();
-  }, []);
+  }, [accountId, dispatch, fetchTransactions]);
 
   // Load categories if necessary.
   const categories = useCategories();
 
-  const updateSearchQuery = debounce(() => {
-    if (searchText === '' && currentQuery) {
-      updateQuery(currentQuery);
-    } else if (searchText && currentQuery) {
-      updateQuery(
-        queries.makeTransactionSearchQuery(
-          currentQuery,
-          searchText,
-          state.dateFormat,
-        ),
-      );
-    }
-  }, 150);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏what arg are we missing in the deps here?

Reply:
Reason for adding that is linter cannot identify proper dependencies of the function due to it being wrapped in `debounce`. Any ideas to get around this?
![image](https://github.com/actualbudget/actual/assets/20313680/2674a07b-d2c8-4bcd-8818-e8226cc2cbf3)

Reply:
I think easiest solution would be to install and use something like this: https://usehooks.com/usedebounce

But with this we're venturing a bit outside the scope of the PR.. so feel free to do with this comment as you please ;)